recipes/valgrind/files/arm-cortex-support.patch

```diff
@@ -0,0 +1,29 @@
+Valgrind's configure script checks for ${host_cpu} starting with
+armv7, but we call it just arm. Since Cortex A chips are (at least)
+ARMv7, make the configure script recognize arm-cortexa*.
+
+Origin: OE-lite (first introduced in 5b66593c)
+Upstream-status: Inappropriate
```

Comment:
Hm, yocto does something different, which doesn't require a patch but is a little hacky:
```
# valgrind checks host_cpu "armv7*)", so we need to over-ride the autotools.bbclass default --host option
EXTRA_OECONF_append_arm = " --host=armv7${HOST_VENDOR}-${HOST_OS}"
```
Not sure what the best solution really is. But I'll change the Upstream-status appropriately if I stick with the patch.